README.md

```diff
@@ -1,2 +1,345 @@
+INFO  - Successfully migrated 1000 tables from HADOOP catalog to NESSIE catalog.
+INFO  - Details:
+INFO  - Successfully migrated these tables:
+[foo.tbl-1, foo.tbl-2, bar.tbl-4, bar.tbl-3, …, …,bar.tbl-1000]
+```
+
+Please note that a log file will be created, which will print "successfully migrated table X" for every table migration, 
+and also log any table level failures, if present.
+
+### B.3) executes the migration and out of 1000 tables 10 tables have failed to migrate because of some error. Remaining 990 tables were successfully migrated.
+
+Sample input:
+```shell
+java -jar iceberg-catalog-migrator-cli-0.1.0-SNAPSHOT.jar migrate \
+--source-catalog-type HADOOP \
+--source-catalog-properties warehouse=/tmp/warehouse,type=hadoop \
+--target-catalog-type NESSIE  \
+--target-catalog-properties uri=http://localhost:19120/api/v1,ref=main,warehouse=/tmp/warehouse \
+--stacktrace
+```
+
+Console output will be same as B.2) till summary because even in case of failure,
+all the identified tables will be attempted for migration.
+
+```
+INFO  - Summary:
+INFO  - Successfully migrated 990 tables from HADOOP catalog to NESSIE catalog.
+ERROR - Failed to migrate 10 tables from HADOOP catalog to NESSIE catalog. Please check the `catalog_migration.log` file for the failure reason.
+Failed Identifiers are written to `failed_identifiers.txt`. Retry with that file using the `--identifiers-from-file` option if the failure is because of network/connection timeouts.
+INFO  - Details:
+INFO  - Successfully migrated these tables:
+[foo.tbl-1, foo.tbl-2, bar.tbl-4, bar.tbl-3, …, …,bar.tbl-1000]
+ERROR  - Failed to migrate these tables:
+[bar.tbl-201, foo.tbl-202, …, …,bar.tbl-210]
+```
+
+Please note that a log file will be generated, which will print "successfully migrated table X" for every table migration and log any table-level failures in the `failed_identifiers.txt` file.
+Users can use this file to identify failed tables and search for them in the log, which will contain the exception stacktrace for those 10 tables. 
+This can help users understand why the migration failed. 
+* If the migration of those tables failed with `TableAlreadyExists` exception, users can rename the tables in the source catalog and migrate only those 10 tables using any of the identifier options available in the argument.
+* If the migration of those tables failed with `ConnectionTimeOut` exception, users can retry migrating only those 10 tables using the `--identifiers-from-file` option with the `failed_identifiers.txt` file.
+* If the migration is successful but deletion of some tables form source catalog is failed, summary will mention that these table names were written into the `failed_to_delete.txt` file and logs will capture the failure reason.
+Do not operate these tables from the source catalog and user will have to delete them manually.
+
+### B.4)  executes the migration and out of 1000 tables. But manually aborts the migration by killing the process.
```

Comment:
When the user kills the process or another hard failure occurs, the "result files" will not be populated and the user "left in the dark". I do *NOT* want that to be fixed in this PR - but it *HAS TO BE* fixed.

Reply:
Created https://github.com/projectnessie/iceberg-catalog-migrator/issues/8 for tracking